.github/workflows/prepare-hotfix-release.yml

```diff
@@ -88,6 +88,15 @@ jobs:
           git config --global user.email '32066757+storybook-bot@users.noreply.github.com'
           yarn release:pick-patches
 
+      - name: Cancel when no hotfixes to pick
```

Comment:
Brings back https://github.com/storybookjs/storybook/pull/24274

However this step won't have any effect until this PR has been cherry-picked to `main`, because the `pick-patches.ts` script that sets `pr-count` runs from `main`.

The last null check ensures that when this is not merged to `main`, this will never cancel, to not block a release process.